migrate.go

```diff
@@ -234,6 +234,32 @@ func versionFilter(v, current, target int64) bool {
 	return false
 }
 
+// EnsureDBVersions retrieves all executed versions for this DB.
+// Create and initialize the DB version table if it doesn't exist.
+func EnsureDBVersions(db *sql.DB) (map[int64]bool, error) {
```

Comment:
What's the new func for? Can't we use the already existing EnsureDBVersion()?

Reply:
To perform up-all, we need to check all migrations from you migration directory and compare them with all migrations from your version table. If migration from directory is not in map from function or its value is false, we need to execute it. So this new function returns list of all migrations from your migration table. It returns it in map with version_id as index and is_applied as value.
EnsureDBVersion returns only current db version, it is not for my command. Also I tried not to